scripts/setup_dev_data.sh

```diff
@@ -81,6 +82,16 @@ dd if=/dev/zero of=fake-h5ad-file.h5ad bs=1024 count=1024 &>/dev/null
 ${local_aws} s3 cp fake-h5ad-file.h5ad s3://corpora-data-dev/
 rm fake-h5ad-file.h5ad
 
+# Make a WMG cube
+echo "Writing WMG cube to s3"
+tmp_cube_dir=`mktemp -d`
+# TODO: Also generate & store:
+#  * s3://wmg-<env>/latest_snapshot_uuid
+#  * cell type orderings
+#  * total cell counts cube
+python3 -m tests.unit.backend.wmg.fixtures.cube ${tmp_cube_dir}
+${local_aws} s3 cp --recursive ${tmp_cube_dir} s3://wmg-dev/dummy-snapshot/cube/
```

Comment:
use sync instead of cp --recursive?

Reply:
that would work too! not sure there's an advantage for GHA testing, since it always starts clean, but maybe with local usage, in case localstack state is being preserved and re-init'd? Since I'll have to touch this again in a future story, I'll look into changing it at that time; for now just trying to close the book on this branch.

Reply:
done now on next story branch